src/main/python/call_graphs/call_graph_edges.py

```diff
@@ -1,30 +1,14 @@
-from IPython.display import HTML
 import os
 import sys
 from glob import glob
-import pyan
 from call_graph_highlight import Highlighted_Callgraph
+from statical_call_graph import StaticalCallGraph
 
-files = [fn for fn in glob(sys.argv[1], recursive=True)
-         if 'venv' not in os.path.normpath(fn)]
+project_path = sys.argv[1]
+node = sys.argv[2]
```

Comment:
If this does what I think it does we should not use sys argv. We should call it from the main.py and get the parameters from there. Otherwise it beats the purpose